lms/djangoapps/teams/static/teams/js/views/teams_tab.js

```diff
@@ -17,12 +17,35 @@
                        this.headerView = new HeaderView({
                            model: this.headerModel
                        });
+                       // TODO replace this with actual views!
+                       var TempTabView = Backbone.View.extend({
+                           initialize: function (options) {
+                               this.text = options.text;
+                           },
+
+                           render: function () {
+                               this.$el.text(this.text)
+                           }
+                       });
+                       this.tabbedView = new TabbedView({
+                           tabs: [{
+                               title: gettext('My Teams'),
+                               url: 'teams',
+                               view: new TempTabView({text: gettext('This is the new Teams tab.')})
```

Comment:
Minor nit: I wouldn't put ``gettext`` around these because then they'll end up in the Transifex database.